src/Runner/Hook.php

```diff
@@ -114,12 +114,19 @@ public function run(): void
     {
         $hookConfigs = $this->getHookConfigsToHandle();
 
-        $this->io->write('<comment>' . $this->hook . ':</comment> ');
+        /** @var IO\Base $io */
+        $io = $this->io;
```

Comment:
This is for the benefit of PHPStan, since `getOption()` is not on the `Console\IO` interface.

Reply:
Do we need the getOption method. I think it would be better to handle the option stuff on the outside (inside the Command) and set an explicit setting on the runner with `$runner->setAction` or `$runner->disablePlugins` ...
This way we could get rid of the `option` methods.

Reply:
I'll give this a try. I didn't like adding the `getOption()` method. 🙂